package.json

```diff
@@ -28,6 +28,17 @@
     "engines": {
         "node": ">=4.2.0"
     },
+    "files": [
+        "./bin",
+        "./lib",
+        "!./lib/enu",
+        "./loc",
```

Comment:
Should `loc` be shipped? What is it for?

Reply:
I honestly have no clue; it's already in the package we publish. Maybe that's an oversight too!

Reply:
I did a quick by-hand bisect and loc was first added to the package in 3.7; I have yet to go look why.

Reply:
I don't think it needs to be shipped. It is likely part of the configuration for our localization team. Maybe @sandersn, @csigs, or @pdostalek know.

Reply:
Removing it shaves off 78 files and 2.7 MB (!) unpacked, so removing it would be great.

Reply:
In fact, this folder hasn't been touched in 3 years; does it still contain anything that's in use?

Reply:
After talking in standup, I'm going to send a PR that deletes this folder altogether from the repo.

Reply:
My concern with that is a possible situation where some infra breaks and we stop getting localization updates. But the PR itself is a good place to discuss further.

Reply:
Yeah, for now, this PR keeps the status quo except the limited number of exceptions I listed above.